### ISSUE TITLE
Changement de données region-bretagne

### ISSUE THREAD

**Issue by menbati** [etalab/decp-rama#102]
Bonjour,
La CI s'est arrêtée semble t il à cause de l'appel aux données region-bretagne. 
Il semblerait que le jeu de donnée ait changé : https://data.bretagne.bzh/explore/dataset/decp-crb0/information/?sort=annee

**Comment by ColinMaudry** [comment]
Oui, j'avais vu que la CI était en erreur, mais je n'ai pas le temps de creuser en ce moment. Ma vie perso est assez chargée en ce moment.

Merci ! Je vais essayer de regarder ça bientôt.

**Comment by menbati** [comment]
Hello @ColinMaudry ! Au cas où tu n'aurais pas vu, j'ai tenté une petite PR en espérant qu'elle soit correcte 🤞
